src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Extensions/RangeExtensions.cs

```diff
@@ -16,6 +16,12 @@ internal static class RangeExtensions
             End = new OSharp.Position(-1, -1)
         };
 
+        public static readonly VS.Range UndefinedVSRange = new()
```

Comment:
Given the VS types are mutable we should probably avoid having statics like this OR have the static be a property that re-calculates per-call : `public static readonly VS.Range UndefinedVSRange => new() ...`